### ISSUE TITLE
Sharing Buttons Block - Facebook icon uses wrong color

### ISSUE THREAD

**Issue by ChrisBegley** [Automattic/jetpack#36706]
### Impacted plugin

Jetpack

### Quick summary

The Sharing Buttons Block uses the wrong color for Facebook when it's set to "Icon Only". It's #39579a, their old brand color, but it should be #0866ff: https://about.meta.com/brand/resources/facebook/logo/

### Steps to reproduce

Use the Sharing Buttons Block to add Facebook. Set it to "Icon Only". Notice it's using their old, dark blue instead of their brighter blue as seen here: https://about.meta.com/brand/resources/facebook/logo/

**Comment by jeherve** [comment]
Interestingly, we do use `#0866ff`. It is set here:

https://github.com/Automattic/wp-calypso/blob/905239b385a09fa1794a4ece620c63c8503e3d0e/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss#L303

We then leverage that variable here:

https://github.com/Automattic/jetpack/blob/6fb16714ab8ee5486cfbea7ecbe011cbfdc80324/projects/plugins/jetpack/extensions/blocks/sharing-button/style.scss#L60

And it looks like this in the editor:

<img width="1169" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/0240b836-ae8a-44d3-9caf-6ec302059c18">

Core, however, seems to use an outdated color, `#1778f2`. I opened https://github.com/WordPress/gutenberg/pull/60424 to suggest the change. It's still not the `#39579a` you're seeing though. Could it be that you use another plugin or theme that overwrites that color?

**Comment by ChrisBegley** [comment]
Hi @jeherve , I'm seeing `#39579a` loading in this Jetpack CSS file, which is on my Single Post page: https://katharinemcphee.net/wp-content/plugins/jetpack/_inc/blocks/sharing-button/view.css?minify=false&ver=13.2.2

It's also loading `#39579a` in the Site Editor screen: https://katharinemcphee.net/wp-content/plugins/jetpack/_inc/blocks/editor.css

I'm running Jetpack 13.2.2 and WordPress 6.5.

**Comment by jeherve** [comment]
Ah, thanks for the extra details! I went digging a bit to try to understand what's happening, and found the problem. Although we updated the reference color 6 months ago in https://github.com/Automattic/wp-calypso/pull/83036, we only updated the Jetpack plugin to use that new version 4 weeks ago in #36227. Those changes haven't been released yet, but will be in a few days. We're currently testing Jetpack 13.3 Beta, and that version includes the correct color:
https://plugins.trac.wordpress.org/browser/jetpack/tags/13.3-beta/_inc/blocks/sharing-button/view.css

In short, the problem will be fixed as soon as Jetpack 13.3 is released!

**Comment by ChrisBegley** [comment]
Awesome, thanks for the great detective work!
